src/Psalm/Internal/Analyzer/MethodAnalyzer.php

```diff
@@ -99,13 +99,13 @@ public static function checkStatic(
         CodeLocation $code_location,
         array $suppressed_issues,
         ?bool &$is_dynamic_this_method = false
-    ): bool {
+    ): void {
```

Comment:
why?

Reply:
After my changes, the return value was no longer used.